packages/material-ui/src/Modal/TrapFocus.js

```diff
@@ -61,7 +68,7 @@ function TrapFocus(props) {
         return;
       }
 
-      if (!rootRef.current.contains(doc.activeElement)) {
+      if (rootRef.current && !rootRef.current.contains(doc.activeElement)) {
```

Comment:
Should we warn if rootRef.current is not an instance of `HTMLElement` but the TrapFocus is enabled (like we do in the Popper's prop-types)? It's a broad question, it's no tight to this effort.

Reply:
Sounds good. Probably better than checking for `focus` i.e. duck-typing.